src/Chirp.Razor/Pages/Cheep.cshtml.cs

```diff
@@ -21,17 +21,14 @@ public class CheepModel : PageModel
 
     [BindProperty, Required, StringLength(160)]
     public string CheepMessage { get; set; } = string.Empty; //= string.Empty; fixes null error
-    
  
     public async Task<IActionResult> OnPostAsync()

```

Comment:
Does this function still have to be async, when we don't use the await keyword inside it anymore (same goes for returning the IActionResult wrapped in a Task)?

Reply:
It does yes, it wont work without, i've tested